src/main/resources/langs/es_ES.yml

```diff
@@ -384,6 +384,10 @@ fun:
             - 'Algo me da mala espina'
             - 'Lo dudo mucho'
 
+    PunCommand:
```

Comment:
The language strings should be placed below the Monika command, and above the Random Cat command, just so the language keys are listed in alphabetical order.

Reply:
It should be in the right order now. It seemed to be just the one file.